README.md

```diff
@@ -1,54 +1,83 @@
  
-## What the heck is Jitsi?
-
-Jitsi is your free and open source Zoom, Google Meet, whatnot.  But it is more than that -- it is decentralized, and
-IIRC end-to-end encrypted video conferencing system.  I believe it is used by gather.town many of you got to love.
-Discover more on [wikipedia:Jitsi](https://en.wikipedia.org/wiki/Jitsi).
-
-## Presenters
-
-You can share your poster window.  Jitsi requires browser extension to provide ability to share your desktop. We
-have been use Jitsi happily Chrome/Chromium for years. [Extension link](https://chrome.google.com/webstore/detail/jitsi-meetings/kglhbbefdnlheedjiejgomgmfplipfeb?hl=en-US).
-
-## Visitors
-
-Each poster would open in a new "dedicated" tab or window. Clicking on it again would just lead you to that tab
-/window if it was already open.
-
-## Implementation
-
-This "website" is a crude lobotomization of the basic DataLad datasets navigator 
-`datalad create-sibling --with-ui` creates for you. https://datasets.datalad.org is one
-of such websites.
-
-Also, if you check git history, you will see that we had used [datalad run](http://handbook.datalad.org/en/latest/basics/101-109-rerun.html) to annotate how `posters.json` was produced.
-So you can use `datalad rerun _produce_posters.json` (where `_produce_posters.json` is just a git tag) to regenerate `posters.json` happen you either modified script or original `.tsv`. 
-
-
-## Contributors ✨
+## :boom: What the Heck Is Jitsi?
+<p align=justify>Jitsi is your free and open source Zoom, Google Meet, whatnot... But it is more than that -- it is a decentralized, and IIRC end-to-end encrypted video conferencing system. I believe it is used by the <code>gather.town</code> that many of you got to love. Discover more on <a href="https://en.wikipedia.org/wiki/Jitsi">wikipedia:Jitsi</a>.</p>
+
+## :arrow_forward: Implementation
+<p align=justify>This "website" is a crude lobotomization of what the basic DataLad datasets navigator <code>datalad create-sibling --with-ui</code> creates for you; <a href="https://datasets.datalad.org">https://datasets.datalad.org</a> is one of such websites.</p>
+<p align=justify>Also, if you check the git history, you will see that we have used <a href="http://handbook.datalad.org/en/latest/basics/101-109-rerun.html">datalad run</a> to annotate how <code>posters.json</code> was produced. So you can use <code>datalad rerun _produce_posters.json</code> (where <code>_produce_posters.json</code> is just a git tag) to regenerate <code>posters.json</code> using either a modified script or the original <code>.tsv</code>.</p> 
+
+## :computer: How to Guideline
+### :heavy_check_mark: How to Visit the OHBM2020 Posters Jitsi Rooms? 
+Go to our website: [https://datalad-datasets.github.io/ohbm2020-posters/](https://datalad-datasets.github.io/ohbm2020-posters/) & wait for the page to load.
+<br><br>
+<img src="./img/datalad_OHBM2020Posters_icons.png" width="700"/>
+<br><br>
```

Comment:
no need for `<br>` s, just add new line.  
There is a way to add images as well, as

```
![POSTERS](img/datalad_OHBM2020Posters_icons.png)
```

although without width, which I am not sure we would want it anyways since it might have an effct